installers/rpm-code/codes/aravis.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+aravis_main() {
+    # Not visible in GH tabs: https://github.com/AravisProject/aravis/releases
+    codes_yum_dependencies \
+        cmake \
```

Comment:
Doesn't this come from common? Many of our codes depend on cmake but don't re-list it as a dep.

Reply:
Removed in ba580fcd4a56104ca9c0e4ea760b812a3a6ef43e. I thought about distilling the deps, but this comes from the instructions, so I thought it was easier to just leave it there.